src/cli/cli.cpp

```diff
@@ -0,0 +1,123 @@
+#include "cli.hpp"
```

Comment:
We should add a docstring at the top of each file indicating what this module is used for

Reply:
Done